src/API/Site/Controllers/ResponseFromExceptionTrait.php

```diff
@@ -0,0 +1,36 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\Exception\ExceptionWithResponseData;
+use Exception;
+use WP_REST_Response as Response;
+
+/**
+ * Trait ResponseFromExceptionTrait
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers
+ *
+ * @since x.x.x
+ */
+trait ResponseFromExceptionTrait {
+
+	/**
+	 * Get REST response from an exception.
+	 *
+	 * @param Exception $exception
+	 *
+	 * @return Response
+	 */
+	protected function response_from_exception( Exception $exception ): Response {
+		$code = $exception->getCode();
+		$status = $code && is_numeric( $code ) ? $code : 400;
```

Comment:
Assignment alignment
```suggestion
		$code   = $exception->getCode();
		$status = $code && is_numeric( $code ) ? $code : 400;
```